src/main/resources/digital/slovensko/autogram/ui/gui/idsk.css

```diff
@@ -493,6 +507,98 @@ TextFlow.autogram-body-s {
   -fx-stroke: -autogram-error-colour;
 }
 
+/* Borders */
+.borders {
+  -fx-border-color: black;
+}
+
+/* Tabs */
+.tab .tab-label {
+  -fx-text-fill: -autogram-text-colour;
+  -fx-background-color: #f3f2f1;
+  -fx-border-color: #929191;
+  -fx-cursor: hand;
+  -fx-padding: 10px 13px 10px 13px;
+  -fx-font-size: 1.125em;
+  -fx-font-weight: bold;
+  -fx-border-width: 0 0 0 0;
+}
+
+.tab:selected .tab-label {
+  -fx-background-color: white;
+  -fx-border-width: 1px 1px 0 1px;
+}
+
+/* Choice boxes */
+.choice-box {
+  -fx-background-color: white;
+  -fx-border-color: black;
+  -fx-border-width: 2px;
+  -fx-padding: 5px;
+  -fx-font-size: 20px;
+}
+
+.choice-box .context-menu {
```

Comment:
Toto cele, drzme sa notacie govuk/idsk. Cize najma `autogram-` prefixy.

Reply:
Toto sú classy, čo majú javafx komponenty priamo na sebe. To je ok, nie? Tým pádom nemusíme tým komponentom dávať aj našu classu, ale rovno použijeme natívnu.